wiki/osu!api/pl.md

```diff
@@ -1,15 +1,18 @@
 ---
-outdated: true
-outdated_since: f90e775380f1c7041a62f3d7f5e3e29ba4326dee
+no_native_review: true
 ---
 
 # osu!api
 
 osu! posiada publiczne [API](https://en.wikipedia.org/wiki/API), którego używać mogą zewnętrzne usługi. W przeszłości miały do niego dostęp tylko upoważnione osoby, ale obecnie każdy użytkownik osu! może z niego korzystać.
 
-Dokumentacja API znajduje się na [oficjalnej wiki osu-api](https://github.com/ppy/osu-api/wiki). Obecna wersja API korzysta z prywatnych kluczy do autoryzacji żądań. Możesz ubiegać się o klucz API na [stronie rejestracji aplikacji](https://osu.ppy.sh/p/api) osu!.
+Dokumentacja API znajduje się na [oficjalnej wiki osu-api](https://github.com/ppy/osu-api/wiki). Obecna wersja API korzysta z prywatnych kluczy do autoryzacji żądań. Możesz ubiegać się o klucz API na [stronie rejestracji aplikacji](https://osu.ppy.sh/p/api) osu!. Miej na uwadze, że dokumentacja zawarta na wiki osu!api nie jest w pełni kompletna czy wyczerpująca, a samo API nie jest obecnie rozwijane. 
```

Comment:
`w pełni kompletna` reads a bit tautological ("wet water"-ish).

Also if I'm being pedantic, the original says that the docs *might* be incomplete. So taking both things in consideration this could be something like

```suggestion
Dokumentacja API znajduje się na [oficjalnej wiki osu-api](https://github.com/ppy/osu-api/wiki). Obecna wersja API korzysta z prywatnych kluczy do autoryzacji żądań. Możesz ubiegać się o klucz API na [stronie rejestracji aplikacji](https://osu.ppy.sh/p/api) osu!. Miej na uwadze, że dokumentacja zawarta na wiki osu!api może nie być kompletna lub wyczerpująca, a samo API nie jest obecnie rozwijane.
```